libs/application/core/src/types/ApplicationTypes.ts

```diff
@@ -12,6 +12,7 @@ export enum ApplicationTypes {
   PARTY_LETTER = 'PartyLetter',
   PARTY_APPLICATION = 'PartyApplication',
   INSTITUTION_COLLABORATION = 'InstitutionCollaboration',
+  APPLICATION_FOR_FUNDING_GOVERNMENT_PROJECTS = 'ApplicationForFundingGovernmentProjects',
```

Comment:
I believe the name here shouldn't contain "application"

Reply:
good point ill fix it